From a3997f3dbe98627e06cc67fe601a4898ea80981a Mon Sep 17 00:00:00 2001
From: Martin Pitt <martin.pitt@ubuntu.com>
Date: Tue, 17 May 2011 09:19:18 +0200
Subject: [PATCH 05/18] Remove %File references

In the past PlaylistPath was meant to point to files only (it can still do so),
but modern players support multiple playlists. Back then, "Dir/%File" meant
"any playlist file in Dir/", but "Dir/" is now equivalent to that, and both
Rhythmbox and Banshee now completely ignore "%File".

Remove the now obsolete %File from all mpi files to remove confusion.
---
 media-players/archos_5.mpi                         |    2 +-
 .../cowon_0x0510_0x0520_0x0710_0x0730_0x0750.mpi   |    2 +-
 media-players/cowon_0x0601_0x0602_0x0604.mpi       |    2 +-
 media-players/cowon_iaudio-7.mpi                   |    2 +-
 media-players/cowon_iaudio-9.mpi                   |    2 +-
 media-players/cowon_iaudio-m3.mpi                  |    2 +-
 media-players/cowon_iaudio-u3.mpi                  |    2 +-
 media-players/cowon_iaudio-u5_g2.mpi               |    2 +-
 media-players/cowon_iaudio-x5.mpi                  |    2 +-
 media-players/cowon_j3.mpi                         |    2 +-
 media-players/cowon_s9.mpi                         |    2 +-
 media-players/google_nexus-one.mpi                 |    2 +-
 media-players/htc_desire_hero_evo-4g_legend.mpi    |    2 +-
 media-players/htc_diamond.mpi                      |    2 +-
 media-players/htc_dream_adp1_g1_magic_tattoo.mpi   |    2 +-
 media-players/htc_incredible.mpi                   |    2 +-
 media-players/htc_snap.mpi                         |    2 +-
 media-players/htc_touch-hd.mpi                     |    2 +-
 media-players/htc_vision.mpi                       |    2 +-
 media-players/huawei_pulse.mpi                     |    2 +-
 media-players/insignia_ns-8v24.mpi                 |    2 +-
 media-players/iriver_e100.mpi                      |    2 +-
 media-players/iriver_e30.mpi                       |    2 +-
 media-players/iriver_h320_h340.mpi                 |    2 +-
 media-players/iriver_s10.mpi                       |    2 +-
 media-players/iriver_t30.mpi                       |    2 +-
 media-players/iriver_t60.mpi                       |    2 +-
 media-players/iriver_t7-volcano.mpi                |    2 +-
 media-players/jetflash_0x8004.mpi                  |    2 +-
 media-players/lg_ally_optimus-one_vortex.mpi       |    2 +-
 media-players/meizu-mini_player_m6.mpi             |    2 +-
 media-players/motorola_cliq.mpi                    |    2 +-
 media-players/motorola_droid-2.mpi                 |    2 +-
 media-players/motorola_droid-x.mpi                 |    2 +-
 media-players/motorola_droid_milestone.mpi         |    2 +-
 media-players/nokia_5800.mpi                       |    2 +-
 media-players/nokia_6700.mpi                       |    2 +-
 media-players/nokia_e71.mpi                        |    2 +-
 media-players/nokia_n95.mpi                        |    2 +-
 media-players/nokia_n97.mpi                        |    2 +-
 media-players/nokia_series-60-phones.mpi           |    2 +-
 media-players/philips_gogear-aria.mpi              |    2 +-
 media-players/philips_gogear-opus.mpi              |    2 +-
 media-players/samsung_meizu-m6-miniplayer.mpi      |    2 +-
 media-players/samsung_yepp-yp-m1.mpi               |    2 +-
 media-players/samsung_yepp-yp-p3.mpi               |    2 +-
 media-players/samsung_yp-p2.mpi                    |    2 +-
 media-players/sandisk_sansa-c250.mpi               |    2 +-
 media-players/sandisk_sansa-clip-plus.mpi          |    2 +-
 media-players/sandisk_sansa-clip-v2.mpi            |    2 +-
 media-players/sandisk_sansa-clip.mpi               |    2 +-
 media-players/sandisk_sansa-e200-series-v2.mpi     |    2 +-
 media-players/sandisk_sansa-e200-series.mpi        |    2 +-
 media-players/sandisk_sansa-fuze-plus.mpi          |    2 +-
 media-players/sandisk_sansa-fuze-v2.mpi            |    2 +-
 media-players/sandisk_sansa-fuze.mpi               |    2 +-
 media-players/sandisk_sansa-view.mpi               |    2 +-
 .../sony-ericsson_xperia-x10-mini-pro.mpi          |    2 +-
 media-players/sony-ericsson_xperia-x10-mini.mpi    |    2 +-
 media-players/sony-ericsson_xperia-x10.mpi         |    2 +-
 60 files changed, 60 insertions(+), 60 deletions(-)

diff --git a/media-players/archos_5.mpi b/media-players/archos_5.mpi
index 17ea884..817c7cd 100644
--- a/media-players/archos_5.mpi
+++ b/media-players/archos_5.mpi
@@ -12,5 +12,5 @@ OutputFormats=audio/mpeg;application/ogg;audio/ogg;audio/x-wav;audio/x-ms-wma;au
 Formats=audio/x-mpegurl;audio/x-scpls
 
 [storage]
-PlaylistPath=Playlist/%File
+PlaylistPath=Playlist/
 AudioFolders=Music/
diff --git a/media-players/cowon_0x0510_0x0520_0x0710_0x0730_0x0750.mpi b/media-players/cowon_0x0510_0x0520_0x0710_0x0730_0x0750.mpi
index 13892c7..aec6ce3 100644
--- a/media-players/cowon_0x0510_0x0520_0x0710_0x0730_0x0750.mpi
+++ b/media-players/cowon_0x0510_0x0520_0x0710_0x0730_0x0750.mpi
@@ -12,6 +12,6 @@ OutputFormats=audio/mpeg;application/ogg;audio/x-ms-wma;audio/flac;audio/x-wav
 Formats=audio/x-mpegurl
 
 [storage]
-PlaylistPath=PLAYLIST/%File
+PlaylistPath=PLAYLIST/
 AudioFolders=MUSIC/;RECORD/;VOICE/
 
diff --git a/media-players/cowon_0x0601_0x0602_0x0604.mpi b/media-players/cowon_0x0601_0x0602_0x0604.mpi
index 728127a..5ab7302 100644
--- a/media-players/cowon_0x0601_0x0602_0x0604.mpi
+++ b/media-players/cowon_0x0601_0x0602_0x0604.mpi
@@ -12,5 +12,5 @@ OutputFormats=audio/mpeg;application/ogg;audio/x-ms-wma;audio/x-wav
 Formats=audio/x-mpegurl
 
 [storage]
-PlaylistPath=PLAYLIST/%File
+PlaylistPath=PLAYLIST/
 
diff --git a/media-players/cowon_iaudio-7.mpi b/media-players/cowon_iaudio-7.mpi
index 2364a9d..f0d4cf5 100644
--- a/media-players/cowon_iaudio-7.mpi
+++ b/media-players/cowon_iaudio-7.mpi
@@ -12,5 +12,5 @@ OutputFormats=audio/mpeg;application/ogg;audio/x-ms-wma;audio/flac;audio/x-wav;v
 Formats=audio/x-mpegurl
 
 [storage]
-PlaylistPath=Playlist/%File
+PlaylistPath=Playlist/
 AudioFolders=Music/;Video/;Recordings/FM Radio/;Recordings/Mic/
diff --git a/media-players/cowon_iaudio-9.mpi b/media-players/cowon_iaudio-9.mpi
index 033d82c..0edf02e 100644
--- a/media-players/cowon_iaudio-9.mpi
+++ b/media-players/cowon_iaudio-9.mpi
@@ -12,5 +12,5 @@ OutputFormats=audio/mpeg;application/ogg;audio/x-ms-wma;audio/flac;audio/x-wav;v
 Formats=audio/x-mpegurl
 
 [storage]
-PlaylistPath=Playlist/%File
+PlaylistPath=Playlist/
 AudioFolders=Music/;Video/;Recordings/FM Radio/;Recordings/Mic/
diff --git a/media-players/cowon_iaudio-m3.mpi b/media-players/cowon_iaudio-m3.mpi
index a7a0476..381ea9a 100644
--- a/media-players/cowon_iaudio-m3.mpi
+++ b/media-players/cowon_iaudio-m3.mpi
@@ -12,5 +12,5 @@ OutputFormats=audio/mpeg;application/ogg;audio/x-ms-wma;audio/flac;audio/x-wav
 Formats=audio/x-mpegurl
 
 [storage]
-PlaylistPath=PLAYLIST/%File
+PlaylistPath=PLAYLIST/
 
diff --git a/media-players/cowon_iaudio-u3.mpi b/media-players/cowon_iaudio-u3.mpi
index 8bcaa0a..f2c1dae 100644
--- a/media-players/cowon_iaudio-u3.mpi
+++ b/media-players/cowon_iaudio-u3.mpi
@@ -12,6 +12,6 @@ OutputFormats=audio/mpeg;application/ogg;audio/x-ms-wma;audio/flac;audio/x-wav
 Formats=audio/x-mpegurl
 
 [storage]
-PlaylistPath=PLAYLIST/%File
+PlaylistPath=PLAYLIST/
 AudioFolders=RECORDS/FM/;RECORDS/LINEIN/;RECORDS/VOICE/
 
diff --git a/media-players/cowon_iaudio-u5_g2.mpi b/media-players/cowon_iaudio-u5_g2.mpi
index 3ed1bc5..f2bf461 100644
--- a/media-players/cowon_iaudio-u5_g2.mpi
+++ b/media-players/cowon_iaudio-u5_g2.mpi
@@ -12,6 +12,6 @@ OutputFormats=audio/mpeg;application/ogg;audio/x-ms-wma;audio/x-wav
 Formats=audio/x-mpegurl
 
 [storage]
-PlaylistPath=PLAYLIST/%File
+PlaylistPath=PLAYLIST/
 AudioFolders=MUSIC/;RECORD/;VOICE/
 
diff --git a/media-players/cowon_iaudio-x5.mpi b/media-players/cowon_iaudio-x5.mpi
index 405a9fc..30b4029 100644
--- a/media-players/cowon_iaudio-x5.mpi
+++ b/media-players/cowon_iaudio-x5.mpi
@@ -12,5 +12,5 @@ OutputFormats=audio/mpeg;application/ogg;audio/ogg;audio/flac;audio/x-wav;audio/
 Formats=audio/x-mpegurl;audio/x-scpls
 
 [storage]
-PlaylistPath=PLAYLIST/%File;Playlist/%File
+PlaylistPath=PLAYLIST/;Playlist/
 AudioFolders=MUSIC/;Music/
diff --git a/media-players/cowon_j3.mpi b/media-players/cowon_j3.mpi
index 8b7cbde..57ea6fd 100644
--- a/media-players/cowon_j3.mpi
+++ b/media-players/cowon_j3.mpi
@@ -12,7 +12,7 @@ OutputFormats=audio/mpeg;application/ogg;audio/x-ms-wma;audio/flac;audio/x-wav;v
 Formats=audio/x-mpegurl
 
 [storage]
-PlaylistPath=Playlists/%File
+PlaylistPath=Playlists/
 AudioFolders=Music/;Video/;Recordings/FM Radio/;Recordings/Line-in/;Recordings/Mic/
 
 
diff --git a/media-players/cowon_s9.mpi b/media-players/cowon_s9.mpi
index 0932571..43037a3 100644
--- a/media-players/cowon_s9.mpi
+++ b/media-players/cowon_s9.mpi
@@ -12,7 +12,7 @@ OutputFormats=audio/mpeg;application/ogg;audio/x-ms-wma;audio/flac;audio/x-wav;v
 Formats=audio/x-mpegurl
 
 [storage]
-PlaylistPath=Playlists/%File
+PlaylistPath=Playlists/
 AudioFolders=Music/;Video/;Recordings/FM Radio/;Recordings/Line-in/;Recordings/Mic/
 
 
diff --git a/media-players/google_nexus-one.mpi b/media-players/google_nexus-one.mpi
index 1020a0f..e6187f4 100644
--- a/media-players/google_nexus-one.mpi
+++ b/media-players/google_nexus-one.mpi
@@ -10,6 +10,6 @@ OutputFormats=audio/mpeg;audio/x-ms-wma;audio/3gpp;audio/mp4;audio/aac;audio/vnd
 
 [storage]
 AudioFolders=Music/
-PlaylistPath=Music/Playlists;PLAYLIST/%File;Playlist/%File
+PlaylistPath=Music/Playlists;PLAYLIST/;Playlist/
 
 
diff --git a/media-players/htc_desire_hero_evo-4g_legend.mpi b/media-players/htc_desire_hero_evo-4g_legend.mpi
index 496902e..6978963 100644
--- a/media-players/htc_desire_hero_evo-4g_legend.mpi
+++ b/media-players/htc_desire_hero_evo-4g_legend.mpi
@@ -12,4 +12,4 @@ Formats=audio/mpegurl;audio/x-mpegurl
 
 [storage]
 AudioFolders=Music/;MP3/
-PlaylistPath=Music/Playlists;PLAYLIST/%File;Playlist/%File
+PlaylistPath=Music/Playlists;PLAYLIST/;Playlist/
diff --git a/media-players/htc_diamond.mpi b/media-players/htc_diamond.mpi
index d248b3e..3d23098 100644
--- a/media-players/htc_diamond.mpi
+++ b/media-players/htc_diamond.mpi
@@ -9,5 +9,5 @@ OutputFormats=audio/mpeg;audio/x-ms-wma;audio/3gpp;audio/mp4;audio/aac;audio/vnd
 
 [storage]
 AudioFolders=Music/
-PlaylistPath=Music/Playlists;PLAYLIST/%File;Playlist/%File
+PlaylistPath=Music/Playlists;PLAYLIST/;Playlist/
 
diff --git a/media-players/htc_dream_adp1_g1_magic_tattoo.mpi b/media-players/htc_dream_adp1_g1_magic_tattoo.mpi
index d26bc3d..d2926c7 100644
--- a/media-players/htc_dream_adp1_g1_magic_tattoo.mpi
+++ b/media-players/htc_dream_adp1_g1_magic_tattoo.mpi
@@ -10,5 +10,5 @@ OutputFormats=audio/mpeg;audio/x-ms-wma;audio/mp4;audio/aac;audio/x-wav;applicat
 
 [storage]
 AudioFolders=Music/
-PlaylistPath=Music/Playlists;PLAYLIST/%File;Playlist/%File
+PlaylistPath=Music/Playlists;PLAYLIST/;Playlist/
 
diff --git a/media-players/htc_incredible.mpi b/media-players/htc_incredible.mpi
index 4c06dd6..76ffb6c 100644
--- a/media-players/htc_incredible.mpi
+++ b/media-players/htc_incredible.mpi
@@ -9,5 +9,5 @@ OutputFormats=audio/mpeg;audio/x-ms-wma;audio/3gpp;audio/mp4;audio/aac;audio/vnd
 
 [storage]
 AudioFolders=Music/
-PlaylistPath=Music/Playlists;PLAYLIST/%File;Playlist/%File
+PlaylistPath=Music/Playlists;PLAYLIST/;Playlist/
 
diff --git a/media-players/htc_snap.mpi b/media-players/htc_snap.mpi
index 713ba9f..42bcc77 100644
--- a/media-players/htc_snap.mpi
+++ b/media-players/htc_snap.mpi
@@ -8,4 +8,4 @@ DeviceMatch=usb:0bb4:0c5f;
 OutputFormats=audio/mpeg;audio/x-ms-wma;audio/aac;audio/x-midi;audio/x-wav
 
 [Storage]
-PlaylistPath=Music/Playlists;PLAYLIST/%File;Playlist/%File
+PlaylistPath=Music/Playlists;PLAYLIST/;Playlist/
diff --git a/media-players/htc_touch-hd.mpi b/media-players/htc_touch-hd.mpi
index 6651fd7..1aaff22 100644
--- a/media-players/htc_touch-hd.mpi
+++ b/media-players/htc_touch-hd.mpi
@@ -9,5 +9,5 @@ OutputFormats=audio/mpeg;audio/x-ms-wma;audio/x-wav
 
 [storage]
 AudioFolders=My Music/
-PlaylistPath=Music/Playlists;PLAYLIST/%File;Playlist/%File
+PlaylistPath=Music/Playlists;PLAYLIST/;Playlist/
 
diff --git a/media-players/htc_vision.mpi b/media-players/htc_vision.mpi
index fb1ea3e..2ef2521 100644
--- a/media-players/htc_vision.mpi
+++ b/media-players/htc_vision.mpi
@@ -9,5 +9,5 @@ OutputFormats=audio/mpeg;audio/x-ms-wma;audio/3gpp;audio/mp4;audio/aac;audio/vnd
 
 [storage]
 AudioFolders=Music/
-PlaylistPath=Music/Playlists;PLAYLIST/%File;Playlist/%File
+PlaylistPath=Music/Playlists;PLAYLIST/;Playlist/
 
diff --git a/media-players/huawei_pulse.mpi b/media-players/huawei_pulse.mpi
index 9657984..1803156 100644
--- a/media-players/huawei_pulse.mpi
+++ b/media-players/huawei_pulse.mpi
@@ -9,4 +9,4 @@ OutputFormats=audio/mpeg;audio/x-ms-wma;audio/3gpp;audio/mp4;audio/aac;audio/vnd
 
 [storage]
 AudioFolders=Music/
-PlaylistPath=Music/Playlists;PLAYLIST/%File;Playlist/%File
+PlaylistPath=Music/Playlists;PLAYLIST/;Playlist/
diff --git a/media-players/insignia_ns-8v24.mpi b/media-players/insignia_ns-8v24.mpi
index cb20b1d..89643a0 100644
--- a/media-players/insignia_ns-8v24.mpi
+++ b/media-players/insignia_ns-8v24.mpi
@@ -12,6 +12,6 @@ OutputFormats=audio/mpeg;application/ogg;audio/x-ms-wma;audio/x-wav;video/mpeg4
 Formats=audio/x-mpegurl
 
 [storage]
-PlaylistPath=playlists/%File
+PlaylistPath=playlists/
 AudioFolders=music/;video/;record/
 
diff --git a/media-players/iriver_e100.mpi b/media-players/iriver_e100.mpi
index 274f867..b9153ce 100644
--- a/media-players/iriver_e100.mpi
+++ b/media-players/iriver_e100.mpi
@@ -12,6 +12,6 @@ OutputFormats=audio/mpeg;audio/x-ms-wma;application/ogg;audio/flac
 Formats=audio/x-iriver-pla
 
 [storage]
-PlaylistPath=Playlists/%File
+PlaylistPath=Playlists/
 AudioFolders=Music/;Recordings/
 
diff --git a/media-players/iriver_e30.mpi b/media-players/iriver_e30.mpi
index 34859a9..88ccffa 100644
--- a/media-players/iriver_e30.mpi
+++ b/media-players/iriver_e30.mpi
@@ -12,6 +12,6 @@ OutputFormats=audio/mpeg;audio/x-ms-wma;application/ogg;audio/flac;audio/x-wav;a
 Formats=audio/x-iriver-pla
 
 [storage]
-PlaylistPath=Playlists/%File
+PlaylistPath=Playlists/
 AudioFolders=Music/;Recordings/Voice;Recordings/FM Radio/;
 
diff --git a/media-players/iriver_h320_h340.mpi b/media-players/iriver_h320_h340.mpi
index de0d5d3..492e39c 100644
--- a/media-players/iriver_h320_h340.mpi
+++ b/media-players/iriver_h320_h340.mpi
@@ -12,6 +12,6 @@ OutputFormats=audio/mpeg;audio/ogg;audio/x-ms-wma;audio/x-wav
 Formats=audio/x-scpls;audio/x-iriver-pla
 
 [storage]
-PlaylistPath=PLAYLIST/%File
+PlaylistPath=PLAYLIST/
 AudioFolders=MUSIC/;RECORD/
 
diff --git a/media-players/iriver_s10.mpi b/media-players/iriver_s10.mpi
index c72140d..1953aa2 100644
--- a/media-players/iriver_s10.mpi
+++ b/media-players/iriver_s10.mpi
@@ -12,6 +12,6 @@ OutputFormats=audio/mpeg;audio/x-ms-wma;application/ogg
 Formats=audio/x-iriver-pla
 
 [storage]
-PlaylistPath=Playlists/%File
+PlaylistPath=Playlists/
 AudioFolders=Music/;Recordings/
 
diff --git a/media-players/iriver_t30.mpi b/media-players/iriver_t30.mpi
index a3fc184..fc49c8b 100644
--- a/media-players/iriver_t30.mpi
+++ b/media-players/iriver_t30.mpi
@@ -12,5 +12,5 @@ OutputFormats=audio/mpeg;audio/x-ms-wma;application/ogg
 Formats=audio/x-iriver-pla
 
 [storage]
-PlaylistPath=Playlists/%File
+PlaylistPath=Playlists/
 
diff --git a/media-players/iriver_t60.mpi b/media-players/iriver_t60.mpi
index 9e6d1aa..1d86ae3 100644
--- a/media-players/iriver_t60.mpi
+++ b/media-players/iriver_t60.mpi
@@ -12,5 +12,5 @@ OutputFormats=audio/mpeg;audio/x-ms-wma;application/ogg
 Formats=audio/x-iriver-pla
 
 [storage]
-PlaylistPath=Playlists/%File
+PlaylistPath=Playlists/
 
diff --git a/media-players/iriver_t7-volcano.mpi b/media-players/iriver_t7-volcano.mpi
index 7886a53..b87d185 100644
--- a/media-players/iriver_t7-volcano.mpi
+++ b/media-players/iriver_t7-volcano.mpi
@@ -12,6 +12,6 @@ OutputFormats=audio/mpeg;application/ogg;audio/x-ms-wma
 Formats=audio/x-iriver-pla
 
 [storage]
-PlaylistPath=Playlists/%File
+PlaylistPath=Playlists/
 AudioFolders=Music/;Recordings/
 
diff --git a/media-players/jetflash_0x8004.mpi b/media-players/jetflash_0x8004.mpi
index c5d859c..de71230 100644
--- a/media-players/jetflash_0x8004.mpi
+++ b/media-players/jetflash_0x8004.mpi
@@ -13,6 +13,6 @@ Formats=audio/x-mpegurl
 
 [storage]
 DriveType=sd_mmc
-PlaylistPath=PLAYLIST/%File
+PlaylistPath=PLAYLIST/
 AudioFolders=/
 
diff --git a/media-players/lg_ally_optimus-one_vortex.mpi b/media-players/lg_ally_optimus-one_vortex.mpi
index e9e060b..526754f 100644
--- a/media-players/lg_ally_optimus-one_vortex.mpi
+++ b/media-players/lg_ally_optimus-one_vortex.mpi
@@ -11,7 +11,7 @@ OutputFormats=audio/mpeg;audio/x-ms-wma;audio/mp4;audio/aac;application/ogg;audi
 Formats=audio/x-mpegurl
 
 [storage]
-PlaylistPath=Playlists/%File
+PlaylistPath=Playlists/
 AudioFolders=Music/;media/audio/
 
 
diff --git a/media-players/meizu-mini_player_m6.mpi b/media-players/meizu-mini_player_m6.mpi
index 27994a9..904db22 100644
--- a/media-players/meizu-mini_player_m6.mpi
+++ b/media-players/meizu-mini_player_m6.mpi
@@ -12,6 +12,6 @@ OutputFormats=audio/mpeg;application/ogg;audio/x-ms-wma;audio/flac;audio/x-wav
 Formats=audio/x-mpegurl
 
 [storage]
-PlaylistPath=PLAYLISTS/%File
+PlaylistPath=PLAYLISTS/
 AudioFolders=MUSIC/;RECORD/
 
diff --git a/media-players/motorola_cliq.mpi b/media-players/motorola_cliq.mpi
index d8d3dff..700b5db 100644
--- a/media-players/motorola_cliq.mpi
+++ b/media-players/motorola_cliq.mpi
@@ -10,5 +10,5 @@ OutputFormats=audio/mpeg;audio/aac;application/ogg
 [storage]
 RequiresEject=true
 AudioFolders=music
-PlaylistPath=Music/Playlists;PLAYLIST/%File;Playlist/%File
+PlaylistPath=Music/Playlists;PLAYLIST/;Playlist/
 FolderDepth=4
diff --git a/media-players/motorola_droid-2.mpi b/media-players/motorola_droid-2.mpi
index 8d0fa7e..fff1ee7 100644
--- a/media-players/motorola_droid-2.mpi
+++ b/media-players/motorola_droid-2.mpi
@@ -9,4 +9,4 @@ Icon=phone-motorola-droid
 OutputFormats=audio/mpeg;audio/aac;audio/x-aac;application/ogg;audio/amr;audio/amr-wb;audio/midi;audio/x-ms-wma
 
 [Storage]
-PlaylistPath=Music/Playlists;PLAYLIST/%File;Playlist/%File
+PlaylistPath=Music/Playlists;PLAYLIST/;Playlist/
diff --git a/media-players/motorola_droid-x.mpi b/media-players/motorola_droid-x.mpi
index 647f81a..1011300 100644
--- a/media-players/motorola_droid-x.mpi
+++ b/media-players/motorola_droid-x.mpi
@@ -10,5 +10,5 @@ OutputFormats=audio/mpeg;audio/aac;application/ogg
 
 [storage]
 AudioFolders=Music/
-PlaylistPath=Music/Playlists;PLAYLIST/%File;Playlist/%File
+PlaylistPath=Music/Playlists;PLAYLIST/;Playlist/
 
diff --git a/media-players/motorola_droid_milestone.mpi b/media-players/motorola_droid_milestone.mpi
index b6c5ae0..a3f8173 100644
--- a/media-players/motorola_droid_milestone.mpi
+++ b/media-players/motorola_droid_milestone.mpi
@@ -11,6 +11,6 @@ OutputFormats=audio/mpeg;audio/aac;application/ogg
 [storage]
 RequiresEject=true
 AudioFolders=Music/
-PlaylistPath=Music/Playlists;PLAYLIST/%File;Playlist/%File
+PlaylistPath=Music/Playlists;PLAYLIST/;Playlist/
 FolderDepth=4
 
diff --git a/media-players/nokia_5800.mpi b/media-players/nokia_5800.mpi
index 45080d7..cfa8f56 100644
--- a/media-players/nokia_5800.mpi
+++ b/media-players/nokia_5800.mpi
@@ -8,6 +8,6 @@ DeviceMatch=usb:0421:0156;
 OutputFormats=audio/mpeg;audio/aac;audio/x-aac;audio/x-ms-wma;audio/m4a
 
 [storage]
-PlaylistPath=Playlists/%File
+PlaylistPath=Playlists/
 AudioFolders=Music/;Sounds/Digital/;MUSIC/
 
diff --git a/media-players/nokia_6700.mpi b/media-players/nokia_6700.mpi
index 30afe10..fb8be3a 100644
--- a/media-players/nokia_6700.mpi
+++ b/media-players/nokia_6700.mpi
@@ -11,6 +11,6 @@ OutputFormats=audio/mpeg;audio/aac;audio/x-aac;audio/x-ms-wma;audio/m4a
 Formats=audio/x-mpegurl
 
 [storage]
-PlaylistPath=Playlists/%File
+PlaylistPath=Playlists/
 AudioFolders=Music/
 
diff --git a/media-players/nokia_e71.mpi b/media-players/nokia_e71.mpi
index a99dd6b..ac3a454 100644
--- a/media-players/nokia_e71.mpi
+++ b/media-players/nokia_e71.mpi
@@ -11,6 +11,6 @@ OutputFormats=audio/mpeg;audio/aac;audio/x-ms-wma;audio/x-wav
 Formats=audio/x-mpegurl
 
 [storage]
-PlaylistPath=Playlists/%File
+PlaylistPath=Playlists/
 AudioFolders=Music/;Music Downloads/;Sounds/Digital/;Sounds/Simple/
 
diff --git a/media-players/nokia_n95.mpi b/media-players/nokia_n95.mpi
index 7019e22..9f44bfb 100644
--- a/media-players/nokia_n95.mpi
+++ b/media-players/nokia_n95.mpi
@@ -12,4 +12,4 @@ Formats=audio/x-mpegurl
 
 [storage]
 AudioFolders=Sounds/Digital;Sounds/
-PlaylistPath=%File
+PlaylistPath=
diff --git a/media-players/nokia_n97.mpi b/media-players/nokia_n97.mpi
index 3af4f8a..0e86fbd 100644
--- a/media-players/nokia_n97.mpi
+++ b/media-players/nokia_n97.mpi
@@ -12,4 +12,4 @@ Formats=audio/x-mpegurl
 
 [storage]
 AudioFolders=Sounds/Digital
-PlaylistPath=%File
+PlaylistPath=
diff --git a/media-players/nokia_series-60-phones.mpi b/media-players/nokia_series-60-phones.mpi
index a705943..f78353e 100644
--- a/media-players/nokia_series-60-phones.mpi
+++ b/media-players/nokia_series-60-phones.mpi
@@ -13,6 +13,6 @@ Formats=audio/x-mpegurl
 
 [storage]
 RequiresEject=true
-PlaylistPath=Playlists/%File
+PlaylistPath=Playlists/
 AudioFolders=Music/;Music Downloads/;Sounds/Digital/;Sounds/Simple/
 
diff --git a/media-players/philips_gogear-aria.mpi b/media-players/philips_gogear-aria.mpi
index 26eb927..c6716c7 100644
--- a/media-players/philips_gogear-aria.mpi
+++ b/media-players/philips_gogear-aria.mpi
@@ -11,5 +11,5 @@ OutputFormats=audio/mpeg;audio/x-ms-wma;audio/x-flac;application/ogg
 Formats=audio/x-mpegurl
 
 [storage]
-PlaylistPath=Playlists/%File
+PlaylistPath=Playlists/
 AudioFolders=Music/;Video/
diff --git a/media-players/philips_gogear-opus.mpi b/media-players/philips_gogear-opus.mpi
index 1fb2b21..77658ed 100644
--- a/media-players/philips_gogear-opus.mpi
+++ b/media-players/philips_gogear-opus.mpi
@@ -12,5 +12,5 @@ OutputFormats=audio/mpeg;audio/x-ms-wma;audio/x-flac;audio/aac
 Formats=audio/x-mpegurl
 
 [storage]
-PlaylistPath=Playlists/%File
+PlaylistPath=Playlists/
 AudioFolders=Music/;Video/;RECORD/
diff --git a/media-players/samsung_meizu-m6-miniplayer.mpi b/media-players/samsung_meizu-m6-miniplayer.mpi
index 40d740d..a343991 100644
--- a/media-players/samsung_meizu-m6-miniplayer.mpi
+++ b/media-players/samsung_meizu-m6-miniplayer.mpi
@@ -12,6 +12,6 @@ OutputFormats=audio/mpeg;audio/ogg;audio/x-ms-wma;audio/flac;audio/x-wav
 Formats=audio/x-mpegurl
 
 [storage]
-PlaylistPath=PLAYLISTS/%File
+PlaylistPath=PLAYLISTS/
 AudioFolders=MUSIC;RECORD
 
diff --git a/media-players/samsung_yepp-yp-m1.mpi b/media-players/samsung_yepp-yp-m1.mpi
index 9bd438f..d4ade43 100644
--- a/media-players/samsung_yepp-yp-m1.mpi
+++ b/media-players/samsung_yepp-yp-m1.mpi
@@ -9,6 +9,6 @@ InputFormats=audio/mpeg
 OutputFormats=audio/mpeg;audio/x-ms-wma;application/ogg;audio/aac;audio/flac;image/jpeg;text/plain;application/x-shockwave-flash;video/x-ms-wmv
 
 [storage]
-PlaylistPath=Playlists/%File
+PlaylistPath=Playlists/
 AudioFolders=Music/;Video/;Datacasts/;Recorded Files/FM Radio/;Recorded Files/Voice/
 
diff --git a/media-players/samsung_yepp-yp-p3.mpi b/media-players/samsung_yepp-yp-p3.mpi
index 3ce1488..ced923e 100644
--- a/media-players/samsung_yepp-yp-p3.mpi
+++ b/media-players/samsung_yepp-yp-p3.mpi
@@ -9,6 +9,6 @@ InputFormats=audio/mpeg
 OutputFormats=audio/mpeg;audio/x-ms-wma;application/ogg;audio/aac;audio/flac;image/jpeg;text/plain;application/x-shockwave-flash;video/x-ms-wmv
 
 [storage]
-PlaylistPath=Playlists/%File
+PlaylistPath=Playlists/
 AudioFolders=Music/;Video/;Datacasts/;Recorded Files/FM Radio/;Recorded Files/Voice/
 
diff --git a/media-players/samsung_yp-p2.mpi b/media-players/samsung_yp-p2.mpi
index ccc3433..e1ba69d 100644
--- a/media-players/samsung_yp-p2.mpi
+++ b/media-players/samsung_yp-p2.mpi
@@ -9,5 +9,5 @@ InputFormats=audio/x-wav
 OutputFormats=audio/mpeg;audio/x-ms-wma;application/ogg;audio/x-wav;audio/aac
 
 [storage]
-PlaylistPath=Playlists/%File
+PlaylistPath=Playlists/
 AudioFolders=Music/;Video/;Datacasts/;Recorded Files/FM Radio/;Recorded Files/Voice/
diff --git a/media-players/sandisk_sansa-c250.mpi b/media-players/sandisk_sansa-c250.mpi
index bfd1ec9..ed0cbe4 100644
--- a/media-players/sandisk_sansa-c250.mpi
+++ b/media-players/sandisk_sansa-c250.mpi
@@ -10,6 +10,6 @@ OutputFormats=audio/mpeg;audio/x-ms-wma;audio/x-wav
 
 [storage]
 RequiresEject=true
-PlaylistPath=PLAYLIST/%File
+PlaylistPath=PLAYLIST/
 AudioFolders=MUSIC/
 
diff --git a/media-players/sandisk_sansa-clip-plus.mpi b/media-players/sandisk_sansa-clip-plus.mpi
index 40d15b9..18063d1 100644
--- a/media-players/sandisk_sansa-clip-plus.mpi
+++ b/media-players/sandisk_sansa-clip-plus.mpi
@@ -13,6 +13,6 @@ Formats=audio/x-mpegurl
 
 [storage]
 RequiresEject=true
-PlaylistPath=%File
+PlaylistPath=
 AudioFolders=MUSIC/;PODCAST/;RECORD/FM/;RECORD/VOICE/;AUDIBLE/;AUDIOBOOKS/
 
diff --git a/media-players/sandisk_sansa-clip-v2.mpi b/media-players/sandisk_sansa-clip-v2.mpi
index 51b959e..d74e339 100644
--- a/media-players/sandisk_sansa-clip-v2.mpi
+++ b/media-players/sandisk_sansa-clip-v2.mpi
@@ -13,6 +13,6 @@ Formats=audio/x-mpegurl
 
 [storage]
 RequiresEject=true
-PlaylistPath=%File
+PlaylistPath=
 AudioFolders=MUSIC/;PODCAST/;RECORD/FM/;RECORD/VOICE/;AUDIBLE/;AUDIOBOOKS/
 
diff --git a/media-players/sandisk_sansa-clip.mpi b/media-players/sandisk_sansa-clip.mpi
index df9d8dd..e9a320e 100644
--- a/media-players/sandisk_sansa-clip.mpi
+++ b/media-players/sandisk_sansa-clip.mpi
@@ -13,6 +13,6 @@ Formats=audio/x-mpegurl
 
 [storage]
 RequiresEject=true
-PlaylistPath=%File
+PlaylistPath=
 AudioFolders=MUSIC/;PODCAST/;RECORD/FM/;RECORD/VOICE/;AUDIBLE/;AUDIOBOOKS/
 
diff --git a/media-players/sandisk_sansa-e200-series-v2.mpi b/media-players/sandisk_sansa-e200-series-v2.mpi
index 4785f0a..61f27e8 100644
--- a/media-players/sandisk_sansa-e200-series-v2.mpi
+++ b/media-players/sandisk_sansa-e200-series-v2.mpi
@@ -12,6 +12,6 @@ Formats=audio/x-mpegurl
 
 [storage]
 RequiresEject=true
-PlaylistPath=%File
+PlaylistPath=
 AudioFolders=MUSIC/;PODCAST/;RECORD/FM/;RECORD/VOICE/;AUDIBLE/;AUDIOBOOKS/
 
diff --git a/media-players/sandisk_sansa-e200-series.mpi b/media-players/sandisk_sansa-e200-series.mpi
index 7ccde7f..11f26db 100644
--- a/media-players/sandisk_sansa-e200-series.mpi
+++ b/media-players/sandisk_sansa-e200-series.mpi
@@ -12,6 +12,6 @@ Formats=audio/x-mpegurl
 
 [storage]
 RequiresEject=true
-PlaylistPath=%File
+PlaylistPath=
 AudioFolders=MUSIC/;PODCAST/;RECORD/FM/;RECORD/VOICE/;AUDIBLE/;AUDIOBOOKS/
 
diff --git a/media-players/sandisk_sansa-fuze-plus.mpi b/media-players/sandisk_sansa-fuze-plus.mpi
index 00673b6..2d137aa 100644
--- a/media-players/sandisk_sansa-fuze-plus.mpi
+++ b/media-players/sandisk_sansa-fuze-plus.mpi
@@ -13,5 +13,5 @@ Formats=audio/x-mpegurl
 
 [storage]
 RequiresEject=true
-PlaylistPath=MUSIC/%File
+PlaylistPath=MUSIC/
 AudioFolders=MUSIC/;PODCAST/;RECORDING/RADIO/;RECORDING/VOICE/;AUDIBLE/;AUDIOBOOKS/
diff --git a/media-players/sandisk_sansa-fuze-v2.mpi b/media-players/sandisk_sansa-fuze-v2.mpi
index 0e1c259..4c0e759 100644
--- a/media-players/sandisk_sansa-fuze-v2.mpi
+++ b/media-players/sandisk_sansa-fuze-v2.mpi
@@ -13,6 +13,6 @@ Formats=audio/x-mpegurl
 
 [storage]
 RequiresEject=true
-PlaylistPath=%File
+PlaylistPath=
 AudioFolders=MUSIC/;PODCAST/;RECORD/FM/;RECORD/VOICE/;AUDIBLE/;AUDIOBOOKS/
 
diff --git a/media-players/sandisk_sansa-fuze.mpi b/media-players/sandisk_sansa-fuze.mpi
index 0989108..6769e47 100644
--- a/media-players/sandisk_sansa-fuze.mpi
+++ b/media-players/sandisk_sansa-fuze.mpi
@@ -13,6 +13,6 @@ Formats=audio/x-mpegurl
 
 [storage]
 RequiresEject=true
-PlaylistPath=%File
+PlaylistPath=
 AudioFolders=MUSIC/;PODCAST/;RECORD/FM/;RECORD/VOICE/;AUDIBLE/;AUDIOBOOKS/
 
diff --git a/media-players/sandisk_sansa-view.mpi b/media-players/sandisk_sansa-view.mpi
index a869ba7..d7257e8 100644
--- a/media-players/sandisk_sansa-view.mpi
+++ b/media-players/sandisk_sansa-view.mpi
@@ -8,6 +8,6 @@ DeviceMatch=usb:0781:74b1;
 OutputFormats=audio/mpeg;audio/x-ms-wma
 
 [storage]
-PlaylistPath=PLAYLIST/%File
+PlaylistPath=PLAYLIST/
 AudioFolders=MUSIC/
 FolderDepth=2
diff --git a/media-players/sony-ericsson_xperia-x10-mini-pro.mpi b/media-players/sony-ericsson_xperia-x10-mini-pro.mpi
index e1c3542..f3310b6 100644
--- a/media-players/sony-ericsson_xperia-x10-mini-pro.mpi
+++ b/media-players/sony-ericsson_xperia-x10-mini-pro.mpi
@@ -12,4 +12,4 @@ Formats=audio/mpegurl;audio/x-mpegurl;audio/x-scpls
 
 [storage]
 AudioFolders=Music/;music/
-PlaylistPath=Music/Playlists;PLAYLIST/%File;Playlist/%File
+PlaylistPath=Music/Playlists;PLAYLIST/;Playlist/
diff --git a/media-players/sony-ericsson_xperia-x10-mini.mpi b/media-players/sony-ericsson_xperia-x10-mini.mpi
index fe2ad37..61b8de3 100644
--- a/media-players/sony-ericsson_xperia-x10-mini.mpi
+++ b/media-players/sony-ericsson_xperia-x10-mini.mpi
@@ -9,7 +9,7 @@ OutputFormats=audio/mpeg;audio/x-ms-wma;audio/3gpp;audio/mp4;audio/aac;audio/vnd
 
 [storage]
 AudioFolders=Music/
-PlaylistPath=Music/Playlists;PLAYLIST/%File;Playlist/%File
+PlaylistPath=Music/Playlists;PLAYLIST/;Playlist/
 
 [Playlist]
 Formats=audio/mpegurl;audio/x-mpegurl;audio/x-scpls
diff --git a/media-players/sony-ericsson_xperia-x10.mpi b/media-players/sony-ericsson_xperia-x10.mpi
index 5556daf..2087e5b 100644
--- a/media-players/sony-ericsson_xperia-x10.mpi
+++ b/media-players/sony-ericsson_xperia-x10.mpi
@@ -9,7 +9,7 @@ OutputFormats=audio/mpeg;audio/x-ms-wma;audio/3gpp;audio/mp4;audio/aac;audio/vnd
 
 [storage]
 AudioFolders=Music/
-PlaylistPath=Music/Playlists;PLAYLIST/%File;Playlist/%File
+PlaylistPath=Music/Playlists;PLAYLIST/;Playlist/
 
 [Playlist]
 Formats=audio/mpegurl;audio/x-mpegurl;audio/x-scpls
-- 
1.7.6

